kurl_util/deploy/Dockerfile

```diff
@@ -12,7 +14,7 @@ RUN make build/bin
 
 FROM ubuntu:jammy
 
-RUN apt-get update -qq && apt-get install -y --no-install-recommends \
+RUN apt-get update -qq && apt-get install -y --no-install-recommends tzdata \
```

Comment:
since this is a multi-stage build do you need the env in the second stage as well?